sass/px-dropdown.scss

```diff
@@ -29,21 +25,21 @@ $inuit-enable-btn--disabled: true;
 $inuit-enable-btn--bare: true;
 $inuit-enable-btn--bare--primary: true;
 @import "px-buttons-design/_objects.buttons.scss";
+@import "px-shadows-design/_objects.shadows.scss";
 
 // Component
 :host {
   display: flex;
   position: relative;
-  text-align: left;
-  -webkit-tap-highlight-color: rgba(0,0,0,0);
+  // -webkit-tap-highlight-color: rgba(0,0,0,0);// <-- Why is this defined twice
```

Comment:
Using `outline: none` is an accessibility issue, because a keyboard user can't tell where their focus is. (I just tested this on px-dropdown, and I had no idea when I was focused on the dropdown.)

It's fine to use `outline: none` if we implement some alternative way of showing the user the currently focused element when navigating with the keyboard. Rob Dodson suggests using the [focus-ring polyfill](https://github.com/WICG/focus-ring).